src/soundio/sounddevicenetwork.cpp

```diff
@@ -267,3 +293,101 @@ void SoundDeviceNetwork::writeProcess() {
         m_pNetworkStream->writingDone(copyCount);
     }
 }
+
+void SoundDeviceNetwork::callbackProcessClkRef() {
+    // This must be the very first call, to measure an exact value
+    updateCallbackEntryToDacTime();
+
+    Trace trace("SoundDeviceNetwork::callbackProcessClkRef %1",
+                getInternalName());
+
+
+    if (!m_denormals) {
+        m_denormals = true;
+        // This disables the denormals calculations, to avoid a
+        // performance penalty of ~20
+        // https://bugs.launchpad.net/mixxx/+bug/1404401
+#ifdef __SSE__
+        if (!_MM_GET_DENORMALS_ZERO_MODE()) {
+            qDebug() << "SSE: Enabling denormals to zero mode";
+            _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);
+        } else {
+             qDebug() << "SSE: Denormals to zero mode already enabled";
+        }
+
+        if (!_MM_GET_FLUSH_ZERO_MODE()) {
+            qDebug() << "SSE: Enabling flush to zero mode";
+            _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
+        } else {
+             qDebug() << "SSE: Flush to zero mode already enabled";
+        }
+        // verify if flush to zero or denormals to zero works
+        // test passes if one of the two flag is set.
+        volatile double doubleMin = DBL_MIN; // the smallest normalized double
+        VERIFY_OR_DEBUG_ASSERT(doubleMin / 2 == 0.0) {
+            qWarning() << "SSE: Denormals to zero mode is not working. EQs and effects may suffer high CPU load";
+        } else {
+            qDebug() << "SSE: Denormals to zero mode is working";
+        }
+#else
+        qWarning() << "No SSE: No denormals to zero mode available. EQs and effects may suffer high CPU load";
```

Comment:
We should explicitly document this in the manual. A user did recently [run into an issue with this on an old computer](https://mixxx.org/forums/viewtopic.php?f=3&t=9285).

Reply:
Yes! I have filed an issue: https://github.com/mixxxdj/manual/issues/62